src/app/index.tsx

```diff
@@ -91,6 +92,8 @@ export default (options: AppOptions) => {
 
   const middleware: Middleware[] = [
     navigation.createMiddleware(routes, history),
+    Sentry.initializeWithMiddleware(),
```

Comment:
I don't think so, but maybe you see something I don't?  It runs fine in the specs at least.

Reply:
can you try running `REACT_APP_ENV=test yarn serve:static` to see if pre-rendering works?

Reply:
guessing you meant `start-static`, since that's the only command that has `static` in it.  I ran:
`REACT_APP_ENV=test yarn run start:static`
without any issues and then loaded the site from http://localhost:62873/

Reply:
yea sorry thats right, ok cool.